apps/log/lib/log/model/log.ex

```diff
@@ -61,7 +61,7 @@ defmodule Helix.Log.Model.Log do
     %__MODULE__{}
     |> cast(params, @creation_fields)
     |> validate_required(@required_fields)
-    |> put_primary_key()
+    |> put_change(:log_id, PK.pk_for(__MODULE__))
```

Comment:
Also, about this, on the Ecto way you ideally don't set the fk id fields, you assoc them.

```elixir
record
|> put_assoc(:parent, parent)

vs

record
|> put_change(:parent_id, parent.id) 
```